keanu-project/src/main/java/io/improbable/keanu/vertices/generic/DoubleBayesNetVertex.java

```diff
@@ -0,0 +1,4 @@
+package io.improbable.keanu.vertices.generic;
+
+public class DoubleBayesNetVertex {
```

Comment:
This class does nothing and isn't used?

Reply:
Aha - sorry about this.  This was left around from one of my original prototypes for this feature -I'll get it deleted.